plugins/logs/plugin.go

```diff
@@ -894,6 +927,62 @@ func (p *Plugin) maskEvent(ctx context.Context, txn storage.Transaction, event *
 	return nil
 }
 
+func (p *Plugin) dropEvent(ctx context.Context, txn storage.Transaction, event *EventV1) (bool, error) {
+
+	drop, err := func() (rego.PreparedEvalQuery, error) {
+
+		p.dropMutex.Lock()
+		defer p.dropMutex.Unlock()
+
+		if p.drop == nil {
+
+			query := ast.NewBody(ast.NewExpr(ast.NewTerm(p.config.dropDecisionRef)))
+
+			r := rego.New(
```

Comment:
We should probably pass the `rego.InterQueryBuiltinCache` here too, or else I'm sure that will be requested later :)